sys/include/net/coap.h

```diff
@@ -156,21 +156,62 @@ extern "C" {
 #define COAP_OBS_DEREGISTER      (1)
 /** @} */
 
+/**
+ * @defgroup net_coap_conf    CoAP compile configurations
+ * @ingroup  net_coap
+ * @ingroup  config
+ * @{
+ */
 /**
  * @name    Timing parameters
+ *
+ * These parameters are defined as configurable in RFC 7252, section 4.8.1.
  * @{
  */
+/**
+ * @brief    Timeout in seconds for a response to a confirmable request
+ *
+ * This value is for the response to the *initial* confirmable message. The
+ * timeout doubles for subsequent retries. To avoid synchronization of resends
+ * across hosts, the actual timeout is chosen randomly between COAP_ACK_TIMEOUT
+ * and (COAP_ACK_TIMEOUT * COAP_RANDOM_FACTOR).
```

Comment:
Add `@ref` before the mentioning of these so they are linked.

Reply:
Fixed in 75cc1f7, and fixed a couple of other macro references as well.